README.md

```diff
@@ -436,11 +436,23 @@ setting the `max_attempts` attribute of the `<mentoring>` element.
 
 Before submitting an answer for the first time:
 
-![Max Attempts Before](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/max-attempts-before.png)
+![Max Attempts Before](doc/img/max-attempts-before.png)
 
 After submitting a wrong answer two times:
 
-![Max Attempts Reached](https://raw.githubusercontent.com/edx-solutions/xblock-mentoring/1fb5e3ece6f34b6cf33c956a4fba1d7cbb7349a2/doc/img/max-attempts-reached.png)
+![Max Attempts Reached](doc/img/max-attempts-reached.png)
+
+### Extended Feedback
+
+For a more detailed feedback, the mentoring tag can have the attribute `extended-feedback="true"`.
```

Comment:
This should be `extended_feedback="true"` rather than `extended-feedback="true"`.
I believe extended feedback only works in assessment mode, you might want to mention that.